apps/api/src/test-data/buildEdgeConnectionTestData/buildDualEdgeConnectionTestData/buildOneFromConnectionForInstanceOfEachResourceType.ts

```diff
@@ -44,6 +45,29 @@ const dtosWithoutTypeProperty: DTO<Omit<EdgeConnection, 'type' | 'connectionType
             },
         ],
     },
+    {
+        id: '3002',
+        // Do we want to allow edge connections to a bibliographic Reference?
```

Comment:
I would say yes.  I'm assuming that a digital representation for a bibliographic reference would only be created after the bibliographic reference, so the only thing is that we wouldn't want to have an edge connection to the digital representation, but only to the bibliographic reference.